test/index.js

```diff
@@ -1,7 +1,9 @@
 // This file cannot be written with ECMAScript 2015 because it has to load
 // the Babel require hook to enable ECMAScript 2015 features!
 require('babel-core/register');
-require('babel-polyfill');
+require("babel-core").transform("code", {
+  plugins: ["transform-runtime"]
```

Comment:
If there's something I can do to clarify the intention of this in the code, I'm super happy to add it.